highlight.io/next.config.js

```diff
@@ -9,6 +9,10 @@ const nextConfig = {
 			// by next.js will be dropped. Doesn't make much sense, but how it is
 			fs: false, // the solution
 		}
+		config.module.rules.push({
+			test: /\.snippet\./,
+			use: 'raw-loader',
+		})
```

Comment:
cool, what does this do?

Reply:
Was trying to put the code snippets in actual python files, but I gave up after entering webpack hell. Deleting...